test/fixtures/parser/ansi/truncate_a.yml

```diff
@@ -1,6 +1,6 @@
 file:
   statement:
     truncate_table:
-    - keyword: truncate
-    - table_reference:
+      keyword: truncate
+      table_reference:
```

Comment:
Nit, but is it confusing to make this unrelated change in this PR?

Reply:
This is a cosmetic and unintentional but automated change that comes up occasionally when running `test/generate_parse_fixture_yml.py`. We've tried to set the YAML formatting options in that script so the output is always consistent, but have not yet succeeded in avoiding all such changes. Definitely a nuisance! Got any ideas about how to fix that script? I haven't looked at it in a few months.

Reply:
Yeah that's fine. Wasn't sure what convention was here: keep changes solely related to this PR, or just accept automatic formatting changes here (both options have their merits!), so thought I'd raise.